Releasing.md

```diff
@@ -13,7 +13,7 @@
 3. Commit your update to [History.md](./History.md). Use the message `Release
    notes for v<new version number>`.
 4. Follow the instructions on
-   https://discourse.purescript.org/t/how-i-publish-a-purescript-package/2482.
+   https://github.com/purescript-contrib/governance/blob/main/pursuit-preregistry.md.
```

Comment:
Updated the Releasing guide as suggested by @cprussin in https://github.com/citizennet/purescript-httpure/pull/197#issuecomment-1318838543.